src/Files.App/Data/Contracts/IWindowsWallpaperService.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Contracts
+{
+	public interface IWindowsWallpaperService
```

Comment:
Shouldn't these be tasks?

Reply:
What do you mean by tasks?

Reply:
SetDesktopWallpaper()

Reply:
Still don't understand.
You mean this should be called on startup?

Reply:
Why are they `bool`?

Reply:
I made a mistake in ISecurityService but all methods that use win32 api should return bool without exception when failed. Win32 error code can be retrieved from Marshal.GetPInvokeLastError().
If caller needs error code, it can call that function within the caller and do error handling.

Reply:
I changed my mind. callers catch exceptions and show the error dialog.